README.md

```diff
@@ -68,37 +70,40 @@ We rely on different `django` and `mypy` versions:
 | 0.12.x       | old semantic analyzer (<0.711), dmypy support | 2.1.x | ^3.6
 
 
-## FAQ
+# FAQ
```

Comment:
This should remain a heading level 2, since it's not the main heading for the page


```suggestion
## FAQ
```